src/core/frame.h

```diff
@@ -145,7 +145,9 @@ typedef enum QUIC_FRAME_TYPE {
     QUIC_FRAME_CONNECTION_CLOSE     = 0x1cULL, // to 0x1d
     QUIC_FRAME_CONNECTION_CLOSE_1   = 0x1dULL,
     QUIC_FRAME_HANDSHAKE_DONE       = 0x1eULL,
-    /* 0x1f to 0x2f are unused currently */
+    /* 0x1f to 0x20 are unused currently */ 
+    QUIC_FRAME_RELIABLE_RESET_STREAM= 0x21ULL, // intentionally ignore type 0x20 of QUIC_RESET_STREAM cause it's likely to be removed from RFC.
```

Comment:
`QUIC_FRAME_IS_KNOWN` needs to be updated to include this.

Reply:
Done.